doc/import_external_docs.ps1

```diff
@@ -2,15 +2,16 @@ Set-PSDebug -Trace 1
 
 $external_docs =
 @(
-    @("https://github.com/unoplatform/uno.wasm.bootstrap", "uno.wasm.bootstrap", "4abadfc93ffeddc82420cc28af04cd7f6b2693ab"),
-    @("https://github.com/unoplatform/uno.themes", "uno.themes", "3d12f341f3ce9ecd7738e163a3a0904e9b94466f"),
-    @("https://github.com/unoplatform/uno.toolkit.ui", "uno.toolkit.ui", "434712b657f479d1329ff60af3b6f22bb6fdb34c"),
-    @("https://github.com/unoplatform/uno.check", "uno.check", "5dec33b3cb4c26f578c8d6bd7a84000bf265a14e"),
-    @("https://github.com/unoplatform/uno.xamlmerge.task", "uno.xamlmerge.task", "7e8ffef206e87dfea90c53805c45e93a7d8c0b46"),
-    @("https://github.com/unoplatform/figma-docs", "figma-docs", "f13d08f2bd7b62fc274b43a4ede4d75909d0f41f"),
-    @("https://github.com/unoplatform/uno.resizetizer", "uno.resizetizer", "f650e844cb66a7dafa084a7e778f14f4fd3ce8a2"),
-    @("https://github.com/unoplatform/uno.uitest", "uno.uitest", "555453c2985ef2745fe44503c5809a6168d063c2"),
-    @("https://github.com/unoplatform/uno.extensions", "uno.extensions", "767fe7eeb610eb2acadcd2e0f79bc55c43f59f70")
+    # use either commit or branch name (latest commit) in 3rd argument
+    @("https://github.com/unoplatform/uno.wasm.bootstrap", "uno.wasm.bootstrap", "main"),
```

Comment:
I don't think we always want to pull docs from latest `main`/`master`

Reply:
Indeed, we'll always import docs with specific commits. This is done to have more consistent docs w.r.t individual branches and avoid publishing unreleased content.

Reply:
Thanks. I've changed it to a hashtable and allow overriding it by param, which is what the *import_external_docs_test_ script is doing.